### ISSUE TITLE
tensorboard log中关于train_loss值缺少一个乘积

### ISSUE THREAD

**Issue by three-sheep** [wenet-e2e/wenet#1290]
我发现在模型训练时候，终端日志显示的train_loss值与tensorboard中记录的有偏差。
原因是tensorboard记录时，train_loss的loss值缺少一个accum_grad的乘积。
虽然，这不影响模型的训练与结果，但可以稍加改动使其变得更好。
 
问题在wenet/utils/executor.py文件的 
第80行，因为梯度累加的需要train loss 除了一个accum_grad值。
` loss = loss_dict['loss'] / accum_grad`
第89行，直接将这个值写进了tensorboard log，导致tensorboard记录的值不准确 。
` writer.add_scalar('train_loss', loss, self.step)`

应改为：
` writer.add_scalar('train_loss', loss*accum_grad, self.step)`

**Comment by robin1001** [comment]
it have no effect on the final performance. Let's keep it now.
